packages/0x.js/test/utils/fill_scenarios.ts

```diff
@@ -21,6 +26,26 @@ export class FillScenarios {
         this.zrxTokenAddress = zrxTokenAddress;
         this.exchangeContractAddress = exchangeContractAddress;
     }
+    public async initTokenBalancesAsync() {
+        const web3Wrapper = (this.zeroEx as any)._web3Wrapper as Web3Wrapper;
+        for (const token of this.tokens) {
+            if (token.symbol === 'WETH') {
+                // _.noop
+            } else if (token.symbol !== 'ZRX') {
+                const contractInstance = web3Wrapper.getContractInstance(
+                    artifacts.DummyTokenArtifact.abi, token.address,
+                );
+                const dummyToken = new DummyTokenContract(contractInstance, {});
```

Comment:
Is the second arg required? Passing in an empty, anonymous object is a code-smell.

Reply:
It is required. it's the defaults map. If we make it non-required - we'll need a lot of checks in generated code which will blow the bundle size. Will naming it make it better?